api/v1/auxilliary.yaml

```diff
@@ -0,0 +1,59 @@
+---
+openapi: 3.0.2
+info:
+  title: DSSAuxService
+  version: 0.8.0
+  description: Discovery and Synchronization Service Auxilliary API's
+paths:
+  /v1/dss/validate_oauth:
+    summary: Validate Oauth token against the DSS.
+    description: ""
+    get:
+      tags:
+      - dss
+      operationId: validateOauth
+      parameters:
+      - name: owner
+        description: Validate the owner claim matches the provided owner.
+        schema:
+          type: string
+        in: query
+        required: false
+      responses:
+        200:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ValidateOauthResponse'
+        401:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+          description: Bearer access token was not provided in Authorization header,
+            token could not be decoded, or token was invalid.
+        403:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+          description: The access token was decoded successfully but did not include
+            a scope appropriate to this endpoint.
+      security:
+      - AuthFromAuthorizationAuthority:
+      summary: /dss/validate_oauth
+      description: Validate Oauth token against the DSS.
+components:
+  schemas:
+    ValidateOauthResponse:
+      description: Validate Oauth token response
+      type: object
+security:
+- AuthFromAuthorizationAuthority:
+  - dss.read.position_reporting_entities
+  - dss.write.position_reporting_entities
+tags:
+- name: dss
+  description: Auxilliary endpoints exposed by the DSS server.
+x-global-options:
+  go_package: pkg/api/v1//auxpb
```

Comment:
nit: Is this necessary given that this option isn't specified in the RID API and we specify the package explicitly in the Makefile?  It seems like the API spec should be independent of implementation

Reply:
it's not necessary but there's a warning when generating it. SInce this part is specific I figured I'd include it.

Not super necessary for the upstream standards api